server/src/main/java/org/kaazing/gateway/server/config/parse/translate/AddServiceNameVisitor.java

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
This and August2012ToSeptember2012Translator will never be loaded in CE as far as I can tell.  Meaning its essentially dead code for the public.  Probably would be better to move internally.